Makefile

```diff
@@ -68,12 +75,24 @@ test/e2e: $(OPERATOR_SDK)
 		--no-setup \
 		--go-test-flags "$(E2E_TEST_ARGS)"
 
-
 $(local_handler_manifest): deploy/operator.yaml
 	mkdir -p $(dir $@)
 	sed "s#REPLACE_IMAGE#$(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)#" \
 		deploy/operator.yaml > $@
 
+
+$(versioned_operator_manifest): HANDLER_IMAGE_SUFFIX = :$(shell hack/version.sh)
+$(versioned_operator_manifest): version/version.go
```

Comment:
what happens when you have multiple targets with the same name?

Reply:
It's not multiple target this is just adding a Make variable to the target, so for exampe if I change this and one of the dependencie is handler-push it will use the changed HANDLER_IMAGE_SUFFIX

This way we don't have to call "make handler-push" but just add handler-push as a dependency.